compileopts/target.go

```diff
@@ -181,15 +181,28 @@ func LoadTarget(options *Options) (*TargetSpec, error) {
 		case "arm64":
 			llvmarch = "aarch64"
 		case "arm":
-			switch options.GOARM {
-			case "5":
-				llvmarch = "armv5"
-			case "6":
-				llvmarch = "armv6"
-			case "7":
-				llvmarch = "armv7"
-			default:
-				return nil, fmt.Errorf("invalid GOARM=%s, must be 5, 6, or 7", options.GOARM)
+			armOpts := strings.Split(options.GOARM, ",")
+			if len(armOpts) == 1 || len(armOpts) == 2 {
```

Comment:
Can you change this condition to return an early error instead?
See: https://go.dev/wiki/CodeReviewComments#indent-error-flow

Something like this:

```go
if len(armOpts) > 2 {
    // return error here
}
// rest of the code
```

Reply:
Sure, I can change that around